.github/workflows/notarize-immudb4net.yml

```diff
@@ -1,17 +1,15 @@
 name: Notarize immudb4net
 
 on:
-  pull_request:
+  push:
     branches:
       - main
       - develop
-    types:
-      - closed
 
 jobs:
   notarize-repository:
     name: Notarize immudb4net repository with cas and vcn
-    uses: codenotary/notarize-with-cas-and-vcn/.github/workflows/notarize-with-cas-and-vcn.yml@main
+    uses: codenotary/notarize-with-cas-and-vcn/.github/workflows/notarize-with-cas-and-vcn.yml@8808d22b40dbd0257dfb456c1ab5505d1020cc54
```

Comment:
I assume this specific commit of the GH action will be used from now on, right? Would it make sense to have a branch instead?

Reply:
No, because we would then not pin the used action to a specific commit. Please see [Calling a reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow).

Reply:
Maybe we should use a tag for that? It is more readable